CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 ### Changed
 
 - Tests: Bumped `apptest-framework` to latest v1.0.0 release
+- Change E2E tests to pull baseDomain from Cluster values instead of default-apps values
```

Comment:
I'd prefer to not have that in the changelog TBH. For the user (think our customers for example) this isn't relevant.

Reply:
👍  I'll get it removed.

Reply:
What about the previous entries about adding new E2E tests? E.g. https://github.com/giantswarm/ingress-nginx-app/blob/994aeebfe42209dcd4396d527564c000f6100dd6/CHANGELOG.md?plain=1#L12